ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Fragment/InFuture.hs

```diff
@@ -18,6 +18,7 @@ module Ouroboros.Consensus.Fragment.InFuture (
   , defaultClockSkew
     -- ** opaque
   , ClockSkew
+  , unClockSkew
```

Comment:
I'm _pretty_ sure the idea was to prevent constructing a `ClockSkew` the `Num` instance of its payload, since that could easily lead to mistakes about which units are involved.

So I changed "opaque" to say something more along those lines.